README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
+
+My *hobbies* include playing sports such as basketball, baseball, and volleyball. I enjoy the **outdoors** as much as I enjoy spending a lazy day **indoors**. I care deeply about the social, economic, and civil challenges facing our democracy today and I actively donate my time, energy, and resources to many causes I find of pressing importance. I *source* my news at times from [link](https://www.npr.org/) [link](https://www.washingtonpost.com/) or [link](https://thehill.com/) 
+
+#### Lists
+
+* Unordered List
+* Red Fruit
+  + strawberries 
+  + apples 
+  + raspberries 
+  
+  1. Ordered List
+  2. Green Fruit
+    + avocado 
+    + grapes 
+    + pears
+
+image: ![](IMG_6652.JPG)
```

Comment:
I think overall, you've done a good job experimenting with different markdown syntax. Also, thank you for sharing your experience with trying to learn everything from ground zero: as someone coming from the same place, I'd say I relate a lot, and it also took me a while to figure out how to do everything properly. The only issue with this assignment is probably the image, and as Emma commented above, the only thing needed is to upload your image from the desktop to your GitHub repo. Also, I noticed that you only had two commits to this assignment, but I personally had a lot more since I found it easier to keep track of changes (also a good way to help me internalize the Github workflow). I think two commits are okay for this specific assignment, but I'm wondering if it would be better to commit more often when the assignment is more complicated/when it's easier to make mistakes.